sdk/core/logger/test/logger.spec.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as Logger from "../src/logger";
+import * as assert from "assert";
+
+const testLogger = Logger.createClientLogger('test');
+
+describe('AzureLogger', function () {
+  it('is not enabled', () => {
+    Logger.setLogLevel('verbose');
+    assert.ok(!Logger.AzureLogger.enabled);
```

Comment:
I dont understand this.....

We created a test logger and set the log level on it, but it is not enabled?

Reply:
Added a comment to explain why this is what we want.